src/rgw/rgw_main.cc

```diff
@@ -1099,6 +1099,12 @@ int main(int argc, const char **argv)
   }
   r = rgw_perf_start(g_ceph_context);
 
+  r = percentile_perf_start(g_ceph_context);
```

Comment:
This over-rides the earlier 'r' value - arguably that wasn't being checked earlier either and hence there is no regression, but fix both those.

Also this should be called from within rgw_perf_stat itself.